### PR TITLE
LRQA-71125 | master

### PR DESCRIPTION
**Pull request by JingyuanZhao**
https://issues.liferay.com/browse/LRQA-71125
@JoyceWang08211

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStagingWithVersioning.testcase [hunk 1]
```diff
@@ -43,95 +43,109 @@ definition {
...
 
-		Staging.gotoPublishToLive();
+			Staging.gotoPublishToLive();
 
-		Staging.publishToLive(remoteStaging = "true");
+			Staging.publishToLive(remoteStaging = "true");
+		}
 
-		User.logoutPG();
+		task ("Then: Verify Web Content is published to Remote Live and can't edit") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStagingWithVersioning.testcase:130]: Case sensitive:
Then: Verify web content entry is published to remote live and user cannot edit web content entry on remote site.

**Comment by JingyuanZhao** [comment]
ci:test:sf

**Comment by JingyuanZhao** [comment]
ci:test:relevant

**Comment by JingyuanZhao** [comment]
ci:test:stable

**Comment by JingyuanZhao** [comment]
ci:test:staging

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**